docker/composable-sandbox.dockerfile

```diff
@@ -1,6 +1,5 @@
-FROM composablefi/ci-linux:2022-04-18 as builder
+FROM composablefi/ci-linux:2022-08-06 as builder
 
-# ISSUE: we already copied context when started build, what the heck we are roing here? it slows down builds
```

Comment:
we'll get
```
$ make containerize-composable-sandbox
Sending build context to Docker daemon  2.427GB
Step 1/23 : FROM composablefi/ci-linux:2022-08-06 as builder
 ---> 1a4fdb751b4f
Step 2/23 : RUN cargo +nightly build --release -p wasm-optimizer
 ---> Running in 6c40e1519b87
error: could not find `Cargo.toml` in `/` or any parent directory
The command '/bin/bash -o pipefail -c cargo +nightly build --release -p wasm-optimizer' returned a non-zero code: 101
make: *** [Makefile:107: containerize-composable-sandbox] Error 101
```
if remove 2 commands below

Reply:
sure. please delete comment then. thank you. i was wrong. so it kind of limitation. you send context and additionally remount all of it second time by desing of docker.